twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/ui/result/SearchResultActivity.kt

```diff
@@ -74,22 +76,21 @@ class SearchResultActivity : AppCompatActivity(), SearchResultContract.View, Vie
             }
 
             btn_connection_error_action.id -> {
-                presenter.searchTweets(query)
+                presenter.getDataSourceValue()?.invalidate()
             }
         }
     }
 
     override fun showLoader() {
         hideContent()
-        inc_search_result_spinner.visibility = View.GONE
+        inc_search_result_spinner.visibility = View.VISIBLE
     }
 
-    override fun showResult(tweets: MutableList<Status>) {
+    override fun showResult() {
         hideContent()
         rv_search_result_tweets_list.visibility = View.VISIBLE
 
         tweetsAdapter.apply {
```

Comment:
As you are now using the PagedList this `notifyDataSetChanged`()` is probably not needed anymore